### ISSUE TITLE
duplicated Pset ifc importer

### ISSUE THREAD

**Issue by HenkMolhoek** [Autodesk/revit-ifc#569]
When I export a ifc with a Pset I see a part of the Pset 2x times when I open it in BIM360 or Navisworks V2 and V3.
In the IFC, the Pset is not duplicated.

BIM360 and Navisworks 2023 V2 and V3
![BIM360 IFC import](https://user-images.githubusercontent.com/45469583/208199318-3623bfd2-b3c2-4d66-afbc-02127c81cf24.png)

Navisworks 2023 V1
![Navisworks 2023 V1](https://user-images.githubusercontent.com/45469583/208199367-e6125504-d456-48a4-b74b-6236d21ed6d5.png)

Ifc Sample file
[Sample File.zip](https://github.com/Autodesk/revit-ifc/files/10249780/Sample.File.zip)

**Comment by HenkMolhoek** [comment]
What is the status of this issue?

**Comment by HenkMolhoek** [comment]
how can i see duplicates if there are no duplicates in the ifc

**Comment by mrjpeterj** [comment]
What do you think are duplicates ?
Or are you saying that these like `3.4 Revit Category` are not on both a type and an element in the IFC file ?

**Comment by HenkMolhoek** [comment]
yes they are both in the revit file but are they always in both if they are not i can't create a good search set
also
please look at
https://github.com/Autodesk/revit-ifc/issues/568

V2 takes to long and V3 gives strange results.

**Comment by HenkMolhoek** [comment]
is there an update yet?
